happi/tests/test_prompt.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+
+from happi.prompt import enforce_list, read_user_dict
+
+
+def test_user_dict(runner):
```

Comment:
I'm a big fan nowadays of annotating pytest fixtures in test suite arguments. pytest is so magical that at least my configuration has no idea what `runner` is until I mark it `click.testing import CliRunner`.

Also big 👍 on getting additional coverage here.